index.html

```diff
@@ -0,0 +1,148 @@
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Adopt-A-Dog - Homepage</title>
+        <link rel="stylesheet" type="text/css" href="styles/stylesheet.css">
+        <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css?family=Work+Sans|Neuton">
+    </head>
+    <body>
+        <header>
+            <div class="header-logo-checkout-overlay">
+                <div class="header-logo-wrapper">
+                    <img src="images/adoptadog-logo.svg" alt="adopt-a-dog logo">
+                    <h1 class="header-logo-heading">Seattle’s premiere dog adoption agency</h1>
+                </div>
+                <div class="header-checkout-wrapper">
+                    <img class="header-checkout-icon" src="images/cart-icon.svg" alt="checkout cart icon">
+                    <p class="header-checkout-cart">$1234.56</p>
+                    <a href="checkout.html" class="header-checkout-link">Checkout</a>
+                </div>
+            </div>
+            <div class="header-background-image-container">
+                <img class="header-background-image" src="images/header-image.jpg" alt="dog eyes close up">
```

Comment:
I would suggest having the image as part of the header instead of using z index to cover up parts that do not fill the screen. It also needs to fit more of the width, as in the mockups. You will have more direction on how to do this in the next assignment using css instead of html elements.